### DISCUSSION TITLE
GO Midi notes received but no sound

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#592] [Help]
Originally posted 2020-04-03 16:01:10.395000 by pcursaro in GrandOrgue discussion forum on SourceForge<br><br>Fixed.
The Midi Out of one keyboard was damaged, continuously issuing Midi messages fulfilling GO...

Thank you anyway.
Regards.

Dear All,
sorry for boring.

I installed last version of GO with Cracow, St, John Cantius organ samples (Windows 10, 64bit). 
I configured two MIDI keyboards and a MIDI pedal on the channels 1, 2 and 5 of my MIDI interface.
Now, if I click on the keys of the console picture I can ear the organ sounds correctly, so I preume the audio out is ok. 
Midi messages are correctly received (I think) because if I check the Log Midi Events option and I press a key on a MIDI keyboards I can see the messages in the box (see also the attached printscreen) but unfortunately no sound at all.
In other words: audio seems to be ok, MIDI seems to be ok, I can play organ by clicking on the console picture but I cannot ear any sound while playing the MIDI keyboards.

Please, do you have any suggestion?

Thank you so much for your help.

<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962125/thread/2834cf3604/2a62/attachment/GO%20printscreen.jpg<br>

**Comment by larspalo** [comment]
Originally posted 2020-04-03 16:56:09.928000 by rgadrian in GrandOrgue discussion forum on SourceForge<br><br>I would recommend that you do not use a manual setting for your keyboards and pedal, but instead use the &quot;wait for midi event&quot; option in the midi objects co-location. This will automatically configure the program and the problem you are commenting on is more likely to disappear.

**Comment by larspalo** [comment]
Originally posted 2020-04-04 09:21:07.389000 by pcursaro in GrandOrgue discussion forum on SourceForge<br><br>Many thanks Adrián,
sure, I&#039;ll do.

Kind regards.

**Comment by larspalo** [comment]
Originally posted 2020-04-04 10:59:47.424000 by pcursaro in GrandOrgue discussion forum on SourceForge<br><br>Just a final note, in case it could help other GO users.

In the last days while testing different MIDI setups (USB cable and Bluetooth) I got the same error: audio ok, midi ok but no sound at all. After a bit of  desperation  I discovered that reloading the organ fixed the problem.
